plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -943,7 +982,7 @@ public Optional<List<String>> getPartitionNamesByFilter(
             List<String> columnNames,
             TupleDomain<String> partitionKeysFilter)
     {
-        return get(partitionFilterCache, partitionFilter(databaseName, tableName, columnNames, partitionKeysFilter));
+        return getOptional(partitionFilterCache, partitionFilter(databaseName, tableName, columnNames, partitionKeysFilter));
```

Comment:
shouldn't it be also used in bulk loaded cache?

Reply:
perhaps, but I didn't find the bulk loaded cache which caches absence, can you help me with that?